Homework3/task16.py

```diff
@@ -0,0 +1,21 @@
+# Задача 16: Требуется вычислить, сколько раз встречается некоторое число X в массиве A[1..N]. 
+#Пользователь в первой строке вводит натуральное число N – количество элементов в массиве. 
+#В последующих  строках записаны N целых чисел Ai. 
+# Последняя строка содержит число X
+# *Пример:*
+# 5
+#     1 2 3 4 5
+#     3
+#     -> 1
+
+import random
+n = 5
+x = 9
+count = 0
+lst = [random.randint(1,10) for _ in range(n)]
+print(lst)
+for i in range(1, len(lst)):
+    if lst[i] == x:
```

Comment:
Не обязательно использовать функцию `range`, достаточно проходиться по элементам списка.
```suggestion
for i in lst:
    if i == x:
```